rust/agama-lib/src/error.rs

```diff
@@ -33,6 +35,8 @@ pub enum ServiceError {
     UnsuccessfulAction(String),
     #[error("Unknown installation phase: '{0}")]
```

Comment:
```suggestion
    #[error("Unknown installation phase: {0}")]
```
unrelated code but related fix to one below